src/core/Object3D.js

```diff
@@ -921,6 +933,7 @@ class Object3D extends EventDispatcher {
 
 Object3D.DefaultUp = new Vector3( 0, 1, 0 );
 Object3D.DefaultMatrixAutoUpdate = true;
+Object3D.DefaultAutoUpdate = true;
```

Comment:
For consistency...

Should we just call it `DefaultMatrixWorldAutoUpdate`? ... and `this.matixWorldAutoUpdate`?